corehq/apps/users/views/mobile/users.py

```diff
@@ -298,7 +298,11 @@ def page_context(self):
             ),
             'demo_restore_date': naturaltime(demo_restore_date_created(self.editable_user)),
             'group_names': [g.name for g in self.groups],
-            "locations_with_cases": get_locations_with_cases(self.domain, self.editable_user.assigned_location_ids)
+            'locations_with_single_user': get_locations_with_single_user(
+                self.domain,
+                self.editable_user.assigned_location_ids,
```

Comment:
As per our offline discussion, I'll have a look at including cases from relevant child locations if `view_descendands = True`. I will leave out checking ancestor locations, as the relevance of this would be very dependant on the context of a specific project.